### PR TITLE
Change the order of mixed history records

### LINKED ISSUE #3787: Records Order
**Describe the bug**
image
These records show up in the incorrect order. They have the same date circa, but the faster records display lower on the list.

**To Reproduce**
1. Go to https://www.worldcubeassociation.org/results/regions.php?regionId=&eventId=333fm&years=&mixHist=Mixed+History

**Expected behavior**
Fastest times are more at the top?
- This isn't as easy as it sounds. Take for instance If the average was also broken at the same date circa, where would this go in the list? So I'm not for sure on a solution without having it look at the schedule, which is far off in the future if I recall correctly.

### PR DESCRIPTION
**Pull request by Baiqiang**
This fixes #3787

### CODE REVIEW

File: webroot/results/includes/regions_history.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ function showRegionalRecordsHistory () {
   #--- Order: normal history or mixed?
   $order = $chosenHistory
            ? 'event.rank, type, value, year desc, month desc, day desc, roundType.rank desc'
-           : 'year desc, month desc, day desc, roundType.rank desc, event.rank, type, value';
+           : 'year desc, month desc, day desc, event.rank, type, value, roundType.rank desc';
```
Comment (jfly) [inline webroot/results/includes/regions_history.php]: Does this change other orderings at all? I haven't looked through the git history at all, but I wonder if the reason for ranking by `roundType.rank` before `value` was for average results? Like if we changed feet from mean of 3 to average of 5, we might want things to show up in a particular order?

**Comment by Baiqiang** [comment]
@jfly 
Here's the diff. It affects records which were set on the same days.
diff

File: webroot/results/includes/regions_history.php (hunk 1 above)
Comment (Baiqiang) [reply webroot/results/includes/regions_history.php]: It does change something in some case. I think this order should be fine. Both Mo3 and Ao5's type are `average` so it doesn't matter here.

**Comment by jfly** [comment]
Thanks @Baiqiang!! :tada:

**Pull request merged**